contracts/stable_coin_factory/sources/tests/helpers.move

```diff
@@ -57,7 +57,10 @@ module stable_coin_factory::test_helpers {
         let km_storage = test::take_shared<KasaManagerStorage>(test);
         let sk_storage = test::take_shared<SortedKasasStorage>(test);
         let rsc_storage = test::take_shared<RUSDStableCoinStorage>(test);
-        let oracle_holder = return_oracleholder(ctx(test));
+        let oracle_holder = test::take_shared<OracleHolder>(test);
```

Comment:
It's better to move this code to a helper method called `update_oracle_price`.

It can only take the price information and timestamp.

Reply:
Also include the first price update in this method. We do not want to call

"update price to 1800" in every method since the price will always be 1800 when test starts. We only need to change the price in `liquidation` and `redemption` tests.